components/match2/commons/opponent_display.lua

```diff
@@ -127,7 +127,15 @@ function OpponentDisplay.InlineOpponent(props)
 		return opponent.name or ''
 
 	elseif opponent.type == 'solo' then
-		return OpponentDisplay.PlayerInlineOpponent(props)
+		if not opponent.players then
+			return
+		end
+
```

Comment:
`opponent.players` will always be an array for party types. see the type declaration in Module:Opponent